RadarSDK/Radar.m

```diff
@@ -446,15 +456,19 @@ + (void)sendLogConversionRequestWithName:(NSString * _Nonnull) name
 }
 
 + (void)logOpenedAppConversion {
-    // if opened_app has been logged within the last second, don't log it again
-    NSTimeInterval lastAppOpenTimeInterval = [[NSDate date] timeIntervalSinceDate:[RadarSettings lastAppOpenTime]];
-    if (lastAppOpenTimeInterval > 1) {
-        [RadarSettings updateLastAppOpenTime];
-        [self sendLogConversionRequestWithName:@"opened_app" metadata:nil completionHandler:^(RadarStatus status, RadarEvent * _Nullable event) {
-            NSString *message = [NSString stringWithFormat:@"Conversion name = %@: status = %@; event = %@", event.conversionName, [Radar stringForStatus:status], event];
-            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo message:message];
-        }];
-    }
+    // Perform a non-blocking sleep for 1 second before starting
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1 * NSEC_PER_SEC)), dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+        // if opened_app has been logged within the last second, don't log it again
+        NSTimeInterval lastAppOpenTimeInterval = [[NSDate date] timeIntervalSinceDate:[RadarSettings lastAppOpenTime]];
+        
+        if (lastAppOpenTimeInterval > 2) {
+            [RadarSettings updateLastAppOpenTime];
+            [self sendLogConversionRequestWithName:@"opened_app" metadata:nil completionHandler:^(RadarStatus status, RadarEvent * _Nullable event) {
```

Comment:
maybe add an explanatory comment that b/c this isn't from a push notification, metadata is nil